descope/internal/mgmt/accesskey.go

```diff
@@ -50,7 +50,7 @@ func (a *accessKey) SearchAll(ctx context.Context, tenantIDs []string) ([]*desco
 	return unmarshalAccessKeySearchAllResponse(res)
 }
 
-func (a *accessKey) Update(ctx context.Context, id, name string, description *string) (*descope.AccessKeyResponse, error) {
+func (a *accessKey) Update(ctx context.Context, id, name string, description *string, roles []string, keyTenants []*descope.AssociatedTenant, customClaims map[string]any, permittedIPs []string) (*descope.AccessKeyResponse, error) {
```

Comment:
let's change to `tenants`, since for all others attributes we don't mention the "key" prefix.

Reply:
done, also across other places, so we will be consistent